src/systems/ec/implementations/react/templates/pages/event-detail/examples/Default.jsx

```diff
@@ -0,0 +1,815 @@
+                  <Link label="see website" href="/example" />
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Social media
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  {/* :'( Custom social media follow */}
+                  <SocialMediaFollow
+                    className="ecl-u-pa-none ecl-u-bg-white"
+                    links={[
+                      {
+                        href: '/example',
+                        label: 'Twitter',
+                        variant: 'standalone',
+                        iconPosition: 'before',
+                        icon: [
+                          {
+                            shape: 'twitter',
+                            size: 'xl',
+                          },
+                          {
+                            shape: 'twitter_hover',
+                            size: 'xl',
+                          },
+                        ],
+                      },
+                      {
+                        href: '/example',
+                        label: 'Facebook',
+                        variant: 'standalone',
+                        iconPosition: 'before',
+                        icon: [
+                          {
+                            shape: 'facebook',
+                            size: 'xl',
+                          },
+                          {
+                            shape: 'facebook_hover',
+                            size: 'xl',
+                          },
+                        ],
+                      },
+                    ]}
+                  />
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Target audience
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  European Year of Culture Heritage
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Number of seats
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  170 seats, 11 available
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Entrance fee
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">Free</div>
+              </div>
+            </div>
+            <div className="ecl-col-12 ecl-col-md-6">
+              <img
```

Comment:
shouldn't it be a webtools widget here?

Reply:
on a real implementation, yes. In a template... maybe.
but I have no idea where to find information about this widget

Reply:
I tried to add the webtools widget, but to allows it to work we would have to send a request for domain whitelisting (and that's probably not something we want to do for a template)